src/Mono.Android/Android.Runtime/AndroidRuntime.cs

```diff
@@ -225,5 +226,43 @@ protected override IEnumerable<string> GetSimpleReferences (Type type)
 				.Concat (Enumerable.Repeat (j, 1));
 		}
 	}
+
+	class AndroidValueManager : JniRuntime.JniValueManager {
+
+		public override void WaitForGCBridgeProcessing ()
+		{
+			throw new NotImplementedException ();
```

Comment:
This can call `JNIEnv.WaitForBridgeProcessing()`.

Reply:
OK, I will add it. Looks like it is not called yet though.